Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeFormUIModel.swift

```diff
@@ -119,14 +119,17 @@ class CreateSafeFormUIModel {
     }
 
     var isCreateEnabled: Bool {
-        state == .ready &&
-        name != nil && !name!.isEmpty &&
+        (state == .ready || (state == .keyNotFound && userSelectedPaymentMethod == .Relayer)) &&
```

Comment:
why to look for a signer key when execution is done via relayer? `state` related conditions are confusing.

Reply:
The search happens before the user has a chance to override the selected payment method.

Reply:
Do we really need to make sure of this state == .keyNotFound?

Reply:
Without it the create button is not enabled when the user selects relayer execution. We could just ignore any state that is not .ready if `userSelectedPaymentMethod == .Relayer`. But I found that too risky.